docs/user-guide.md

```diff
@@ -200,19 +201,30 @@ When the debugger is attached, the Visual Studio Code [debug options](https://co
 
 ![Debug Mode](/docs/user-guide-screenshots/debugger_mode.png)
 
-## Manually add your Liberty project to the dashboard
+## Manually add your Liberty project to the Liberty dashboard
 
-In the event that your Liberty project is not automatically detected by the Liberty dashboard, you can manually add your Liberty project to the Liberty dashboard. To manually add your Liberty project to the Liberty dashboard, right-click into an empty space in the project explorer and select **Add project to Liberty Dashboard** or select the **Liberty: Add project to the tool window** command in the command palette, followed by your application.
+In the event that your Liberty project is not automatically detected by the Liberty dashboard, you can manually add your Liberty project to the Liberty dashboard in one of four ways:<br>
+1. Right-click into an empty space in the Explorer view and select **Add project to Liberty Dashboard**.<br>
```

Comment:
these shouldn't be steps because the user will just choose one of them, not complete them all in sequence